hack/make/.binary

```diff
@@ -67,6 +81,14 @@ hash_files() {
 				export CC="${CC:-x86_64-linux-gnu-gcc}"
 				export CGO_ENABLED=1
 				;;
+			linux/ppc64le)
+				export CC="${CC:-powerpc64le-linux-gnu-gcc}"
+				export CGO_ENABLED=1
+				;;
+			linux/s390x)
+				export CC="${CC:-s390x-linux-gnu-gcc}"
+				export CGO_ENABLED=1
+				;;
```

Comment:
Yet another. This one I believe we can very safely remove until 22.06.

Reply:
@lmbarros I took the entire file and avoided hacking it. Feel free to remove manually if you like, but I would suggest to keep the file as is, since we don't "own it".